components_new/query_handler/SystemDbReader.py

```
@@ -0,0 +1,126 @@
+    def __init__(self):
+        self.conn = self.get_connection()
+        self.cursor = self.conn.cursor()
+
+    def get_connection(self):
+        
+        isDocker = os.environ.get('IS_RUNNING_IN_DOCKER', False)
+
+        if isDocker:
+            host = "asp-mariadb-container"
+        else:
+            host = "localhost"
```

Ruff (0.8.2)

Use ternary operator `host = 'asp-mariadb-container' if isDocker else 'localhost'` instead of `if`-`else`-block

Replace `if`-`else`-block with `host = 'asp-mariadb-container' if isDocker else 'localhost'`

(SIM108)